### PR TITLE
[BUGFIX] Fix "void result used" warnings

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: api/class.tx_ameosformidable.php [hunk 1]
```diff
@@ -2233,7 +2241,7 @@ public function _render()
                 $this->getJSLoader()->includeBaseLibraries();
 
                 // the datahandler is executed
-                $sDH = $this->getDataHandler()->_doTheMagic(true);
```
Comment (hannesbochmann) [inline api/class.tx_ameosformidable.php]: the $sDH variable is used in line 2587. I think it's not safe to remove the assignment here and in the following changes

**Review by hannesbochmann — CHANGES_REQUESTED** [review]

File: api/class.tx_ameosformidable.php (hunk 1 above)
Comment (oliverklee) [reply api/class.tx_ameosformidable.php]: I'll have a look at it. We probably can just set it to `null` (as the `void` result is just `null`).

Comment (oliverklee) [reply api/class.tx_ameosformidable.php]: Fixed, rebased and repushed. (I've changed `doTheMagic` to always return a string now.)

**Comment by oliverklee** [comment]
I'd like to have #135 merged first so we can see which warnings this change fixes (and that it does not introduce any new problems).

**Pull request closed without merging**